src/components/pages/StudentsPage/WelcomeMessage.css.tsx

```diff
@@ -0,0 +1,18 @@
+import { css } from '@emotion/react';
+
+const unpairedMessageContainer = css`
+  background: #f8e5e0;
+  color: red;
+  width: 98%;
+  max-width: 800px;
+  border: 1px solid white;
+  border-radius: 10px;
+  padding: 70px 0;
```

Comment:
I think the code is cleaner to read when just using padding top and bottom.  Setting the horizontal padding to 0 can confuse whoever is reading the code as padding defaults to 0.
```suggestion
  padding-top: 70px;
  padding-bottom: 70px;
```